src/schema/slack/types/custom/form_input.ts

```diff
@@ -1,8 +1,9 @@
 import SchemaTypes from "../../../schema_types.ts";
 import { DefineType } from "../../../../types/mod.ts";
+import { DefineParameter } from "../../../../parameters/define_parameter.ts";
 
-const FormInput = DefineType({
-  name: "slack#/types/form_input_object",
+const FormInputObject = DefineParameter({
```

Comment:
I think _technically_ we can avoid using `DefineParameter` here as I believe the call to `DefineType` below should function in the same way in terms of providing an entrypoint for the props/required props generics to come into play.

That said, this is all internal stuff not directly exposed to the dev, and I'd rather we use `DefineParameter` to wrap all typed object definitions whenever we can, even if redundant.

Open to either. This same comment applies to all three of the Slack custom types.

Reply:
Good catch, I'd like for us to just use `DefineType` rather than spreading the result of `DefineParameter` into a `DefineType` call, since I think that's what we would recommend to devs.

Reply:
K I updated this in https://github.com/slackapi/deno-slack-sdk/pull/124/commits/26345ff5ce097f1c06efd90fa96dd9713933bd2d and also added a test for the internal `FormInput` slack type to make sure it behaves as we would expect in a function runtime context.